change/@fluentui-eslint-plugin-309e14e3-2c53-4422-b823-9a8d71aa4b0a.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "none",
```

This should still be patch since it technically affects the published package; it's just the dependentChangeType that should be none to avoid tons of extra bumps
```suggestion
  "type": "patch",
```